tasks/run.js

```diff
@@ -73,7 +73,7 @@ module.exports = function (opts) {
 				.then(function() {
 
 					// Silent update — throw away any errors
-					downloadDevelopmentKeys({ update: true });
+					downloadDevelopmentKeys();
```

Comment:
Maybe this doesn't need to be executed at all — it's covered in https://github.com/Financial-Times/next-build-tools/blob/master/tasks/download-development-keys.js